frontend/src/helpers/types/resError.ts

```diff
@@ -1,4 +1,4 @@
 export interface resError {
   error_message: string;
-  additional_information?: { [key: string]: any };
+  additional_information?: { errors?: { loc: string[]; msg: string; type: string }[] };
```

Comment:
why ? did the err format changed on the server ? 
@yromanyshyn please review it also

Reply:
`{
    "error_message": "2 validation errors for Request body -> email str type expected (type=type_error.str) body -> email -> 0 value is not a valid email address (type=value_error.email)",
    "additional_information": {
        "errors": [
            {
                "loc": [
                    "body",
                    "email"
                ],
                "msg": "str type expected",
                "type": "type_error.str"
            },
            {
                "loc": [
                    "body",
                    "email",
                    0
                ],
                "msg": "value is not a valid email address",
                "type": "value_error.email"
            }
        ]
    }
}`

Reply:
"errors": [
            {
                "loc": [
                    "body",
                    "email"
                ],
                "msg": "str type expected",
                "type": "type_error.str"
            },
            {
                "loc": [
                    "body",
                    "email",
                    0
                ],
                "msg": "value is not a valid email address",
                "type": "value_error.email"
            }
        ]
    }
}`

Reply:
`
{
    "error_message": "2 validation errors for Request body -> email str type expected (type=type_error.str) body -> email -> 0 value is not a valid email address (type=value_error.email)",
    "additional_information": {
        "errors": [
            {
                "loc": [
                    "body",
                    "email"
                ],
                "msg": "str type expected",
                "type": "type_error.str"
            },
            {
                "loc": [
                    "body",
                    "email",
                    0
                ],
                "msg": "value is not a valid email address",
                "type": "value_error.email"
            }
        ]
    }
}
`

Reply:
`
{
    "error_message": "2 validation errors for Request body -> email str type expected (type=type_error.str) body -> email -> 0 value is not a valid email address (type=value_error.email)",
    "additional_information": {
        "errors": [
            {
                "loc": [
                    "body",
                    "email"
                ],
                "msg": "str type expected",
                "type": "type_error.str"
            },
            {
                "loc": [
                    "body",
                    "email",
                    0
                ],
                "msg": "value is not a valid email address",
                "type": "value_error.email"
            }
        ]
    }
}
`

Reply:
@karkhypov Please never change global type before you are checking all the endpoints/making sure with the server team if its the same scheme always.

Reply:
Ok, I tried to find it in the `generated.ts` but looks like there is no such scheme there.

Reply:
@yromanyshyn Your assistence please